### ISSUE TITLE
@Default in radio button matrix not working and removing dispayed values

### ISSUE THREAD

**Issue by MathiasAntunes** [susom/redcap-em-shazam#43]
Hi Andy,

I've been playing a lot with Shazam for numerous projects. We encountered a problems with radio buttons in matrix.
We tried to pipe from previous repeating event with @Default="[event-name][field_name][previous-instance]" in a matrix field.

What happens :

- Without @DEFAULT : Shazam works great and answers are stored and displayed
- With @DEFAULT : Answers are stored but not displayed (They appear at the page loading but then they are removed). The @Default functionality seems to work but values are also removed at the end of the page loading.

Design : 
![image](https://user-images.githubusercontent.com/23211270/113717703-2dd03480-96ec-11eb-9459-69d2c5fbf4f6.png)

**Comment by 123andy** [comment]
Yes - the action tags came after shazam and there is probably some competing order here.  If you can send me a simple xml file with an example and then the corresponding shazam setup I can try to debug.  The answer is probably to just add a delay to shazam running until the default javascript task is complete...

-a

________________________________
From: Mathias Antunes ***@***.***>
Sent: Tuesday, April 6, 2021 6:30 AM
To: susom/redcap-em-shazam ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [susom/redcap-em-shazam] @Default in radio button matrix not working and removing dispayed values (#43)


Hi Andy,

I've been playing a lot with Shazam for numerous projects. We encountered a problems with radio buttons in matrix.
We tried to pipe from previous repeating event with @default<https://github.com/default>="[event-name][field_name][previous-instance]" in a matrix field.

What happens :

  *   Without @default<https://github.com/default> : Shazam works great and answers are stored and displayed
  *   With @default<https://github.com/default> : Answers are stored but not displayed (They appear at the page loading but then they are removed). The @default<https://github.com/default> functionality seems to work but values are also removed at the end of the page loading.

Design :
[image]<https://user-images.githubusercontent.com/23211270/113717703-2dd03480-96ec-11eb-9459-69d2c5fbf4f6.png>

**Comment by MathiasAntunes** [comment]
Ok thanks for the quick answer. Here is an example (with and without @default) : 
[shazam_debug.zip](https://github.com/susom/redcap-em-shazam/files/6271125/shazam_debug.zip)

**Comment by 123andy** [comment]
Sorry. @MathiasAntunes  - I haven't had time to get to this...  (and may not for some time) -- so if you can fix it and make a PR that is welcome, otherwise it may be a while...

-A
